4-forms-validations/forms.html

```diff
@@ -0,0 +1,63 @@
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Forms validations</title>
+</head>
+
+<body>
+    <h1>Push-dev form</h1>
+    <form action="">
+        <label for="first_name">
+            First name: <input type="text" id="first_name" name="first_name" required>
+        </label>
+        <label for="last_name">
+            Last name: <input type="text" id="last_name" name="last_name" required>
+        </label>
+        <label for="email">
+            Email: <input type="email" id="email" name="email" required>
+        </label>
+        <label for="date">
+            Birthdate: <input type="date" id="date" name="date" required>
+        </label>
+        <fieldset>
+            <legend>Choose your gender:</legend>
+            <select id="gender">
+                <option value="female">Female</option>
+                <option value="male">Male</option>
+                <option value="other">Other</option>
+            </select>
```

Comment:
```suggestion
          <label for="gender">
          Gender
            <select id="gender">
                <option value="female">Female</option>
                <option value="male">Male</option>
                <option value="other">Other</option>
            </select>
            </label>
```
Please include the label for gender.